crates/rolldown_binding/src/types/binding_output_asset.rs

```diff
@@ -1,17 +1,23 @@
-use derivative::Derivative;
-use serde::Deserialize;
+use napi_derive::napi;
 
-#[napi_derive::napi(object)]
-#[derive(Deserialize, Default, Derivative)]
-#[serde(rename_all = "camelCase")]
-#[derivative(Debug)]
+#[napi]
 pub struct BindingOutputAsset {
-  pub file_name: String,
-  pub source: String,
+  inner: Box<rolldown_common::OutputAsset>,
```

Comment:
```suggestion
  inner: Arc<rolldown_common::OutputAsset>,
```

Considering the above plugin hook, `<Arc>` should be better to avoid O(n) clone overhead before user really access the data.

Reply:
DOne